service/topology_coordinator.cc

```diff
@@ -783,12 +783,15 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                     size_t unimportant_init_tablet_count = 2; // must be a power of 2
                     locator::tablet_map new_tablet_map{unimportant_init_tablet_count};
 
-                    for (const auto& table : ks.metadata()->tables()) {
+                    auto tables_with_mvs = ks.metadata()->tables();
+                    auto views = ks.metadata()->views();
```

Comment:
This will break in the future when we add a third kind of a table. It's sad that we don't consider that view is a table, they're more similar than different. But we can't undo it. We need something like ks.metadata()->tables_of_any_kind().

Reply:
I'm thinking what _a third kind of table_ could be, a virtual table, and index? (Or is an index considered a materialized view already?)
CC @nyh

Reply:
I was refering to future changes, so we don't know what it will be. 

Index is a view.